src/sage/schemes/curves/zariski_vankampen.py

```diff
@@ -1262,7 +1411,7 @@ def conjugate_positive_form(braid):
 
     INPUT:
 
-    - ``braid`` -- a braid ``\sigma``.
+    - ``braid`` -- a braid `\sigma`.
```

Comment:
```suggestion
    - ``braid`` -- a braid `\sigma`
```
since we are fixing stuff.